src/encoded/upgrade/library.py

```diff
@@ -12,3 +12,20 @@ def upgrade_library_1_2(
     if analyte:
         value["analytes"] = [analyte]
         del value["analyte"]
+
+
+@upgrade_step("library", "2", "3")
+def upgrade_library_2_3(
+    value: Dict[str, Any], system: Dict[str, Any]
+) -> Dict[str, Any]:
+    """Delete properties in `removed_properties` from library."""
+
+    removed_properties = [
+        "fragment_minimum_length",
+        "fragment_maximum_length",
+        "fragment_standard_deviation_length",
+        "insert_standard_deviation_length",
+    ]
+    for property in removed_properties:
+        if property in value:
+            del value[property]
```

Comment:
Does it make sense to move the existing `target_insert_mean_length` to the new `target_fragment_size`, and then delete the prior property?

Reply:
That probably makes sense. For non-short read libraries, I'm a little unclear on how submitters will be defining fragments vs. inserts

Reply:
I think that's generally been confusing and is likely the reason people want this simplified to one property.

Reply:
I've changed the upgrader to convert all `target_insert_mean_length`  values to the `target_fragment_size`.